server/api/core/email.js

```diff
@@ -0,0 +1,59 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Job } from "meteor/vsivsi:job-collection";
+import { Jobs, Packages, Templates } from "/lib/collections";
```

Comment:
And in that case, that import is correct.  There is an [Assets collection](https://github.com/reactioncommerce/reaction/blob/master/lib/collections/collections.js#L28-L34), but `Assets.getText` isn't associated.  An unfortunate namespace collision.  You wouldn't be able to use both in the same file without importing the `Assets` collection `as` something else.  
And _that's_ why globals are bad, Meteor.